docs/rules/vue-strong/full-word-component-name.md

```diff
@@ -7,3 +7,31 @@ Checks if the component name is a full word. As this is not easy to detect progr
 
 - **Readability**: Full-word component names are easier to read and understand.
 - **Consistency**: Full-word component names are consistent with the naming of files and folders.
+
+## 😱 Examples of code for which this rule will throw a warning
+
+::: warning
+The following code uses a component name that does not qualify as a full word because it contains fewer than three consonants. This is not in line with best practices for component naming.
+:::
+
+```vue
+<template>
+  <nav>
+    <!-- Navigation items -->
+  </nav>
+</template>
+```
+
+## 🤩 How to fix it?
+
+::: tip
+Refactor the component name to use a full word, ensuring it contains at least three consonants for clarity and better readability:
+:::
+
+```vue
+<template>
+  <navigation>
```

Comment:
Should not be the first letter uppercase?

Reply:
fixed in https://github.com/rrd108/vue-mess-detector/pull/133/commits/443a0b671ee8e8f6bd15f7912d7de6afec1bcf34